src/vaadin-details.html

```diff
@@ -125,12 +125,17 @@
           };
         }
 
+        /**
+         * @return {Element}
```

Comment:
This returns the `div` which has `part="content"` and should always be there (this relies on shadowRoot being ready anyway) so it should be ok to use `HTMLElement` instead to be more explicit and mark the return value as non-nullable.
```suggestion
         * @return {!HTMLElement}
```
results in:
```ts
readonly _collapsible: HTMLElement;
```
(was `Element|null`)

Reply:
Done.